python/seldon_core/storage.py

```diff
@@ -244,22 +244,17 @@ def _download_local(uri, out_dir=None):
 
     @staticmethod
     def _create_minio_client():
-        # Remove possible http scheme for Minio
-        url = urlparse(os.getenv("AWS_ENDPOINT_URL", "s3.amazonaws.com"))
+        # Adding prefixing "http" in urlparse is necessary for it to be the netloc
+        url = urlparse(os.getenv("AWS_ENDPOINT_URL", "http://s3.amazonaws.com"))
         use_ssl = (
             url.scheme == "https"
             if url.scheme
-            # KFServing uses S3_USE_HTTPS, whereas Seldon was already using
-            # USE_SSL.
-            # To keep compatibility with the storage init layer we support
-            # both, giving priority to USE_SSL.
-            # https://github.com/SeldonIO/seldon-core/pull/827
-            # https://github.com/kubeflow/kfserving/pull/362
-            else bool(getenv("USE_SSL", "S3_USE_HTTPS", "false"))
+            else bool(os.getenv("S3_USE_HTTPS", "true"))
```

Comment:
On our docs we document usage of `USE_SSL` (e.g. https://docs.seldon.io/projects/seldon-core/en/latest/servers/overview.html?highlight=USE_SSL#handling-credentials). But KFServing uses `S3_USE_HTTPS` instead. 

Should we add back the removed bit? 

It's a bit of a pain to have that divergence though. Maybe we should consider `USE_SSL` as deprecated and update our docs (and anything on the operator?) to use `S3_USE_HTTPS`. But that's outside the scope of this PR.

Reply:
I've added an issue with the above: https://github.com/SeldonIO/seldon-core/issues/1847

However, in the meantime, we will need to read from both.

Reply:
Oh I see, no I do think you're right, let's probably add both!